test/apps/scroll/test.ts

```diff
@@ -99,7 +99,33 @@ describe('scroll', function() {
 		const secondScrollY = await r.page.evaluate(() => window.scrollY);
 
 		assert.equal(firstScrollY, secondScrollY);
-	});
+  });
```

Comment:
the newly added lines use spaces whereas the existing lines use tabs. should switch to tabs for consistency

Reply:
I've converted all the lines I touched to tabs.